File: youlexuan-cart-interface/src/main/java/com/offcn/cart/service/CartService.java
package com.offcn.cart.service;

import com.offcn.entity.CartVO;

import java.util.List;

public interface CartService {


    /**
     * 添加购物车
     * @param cartList
     * @param itemId
     * @param num
     * @return
     */
    List<CartVO> addGoodsToCartList(List<CartVO> cartList, Long itemId, Integer num);
}

File: youlexuan-cart-web/src/main/webapp/js/service/cartService.js
app.service('cartService',function ($http) {
    this.findCartList = function () {
        return $http.get("../cart/findCartList.do");
    }

    this.addCartList = function (itemId, num) {
        return $http.get("../cart/addCart.do?itemId="+itemId+"&num="+num);
    }
});